### ISSUE TITLE
Cannot write to a bufferedStream while the read buffer is not empty if the underlying stream is not seekable

### ISSUE THREAD

**Issue by pfeiflelukas** [bonesoul/CoiniumServ#832]
Has anyone been able to solve this issue on windows? I have tried different versions of redis and different makes but this problem keeps occurring. Sometimes it happens as soon as i start the program and sometimes it only start 2 hours into running. Does anyone have any ideas??

Thanks

**Comment by Virtuworks** [comment]
Csredis is the issue, we solved all these issues by moving to stackexchange redis client.

**Comment by pfeiflelukas** [comment]
I just installed StackExchange redis and added it to the CoiniumServ solution, but how to i then run that new redis client?

**Comment by NinjaZool** [comment]
Same Error here: _Cannot write to a BufferedStream while the read buffer is not empty if the underlying stream is not seekable. Ensure that the stream underlying this BufferedStream can seek or avoid interleaving read and write operations on this BufferedStream._

I installed StackExchange by uget and after done `find . -name "csredis.dll"` on my home directory, I have copied the dll from uget directory to CoinunmServ directories.

Then i runned CoinunmServ.exe, but after about 3 hours of execution i have the same error again.

My system is Debian 8.
